appinfo/info.xml

```diff
@@ -18,7 +18,7 @@
     </two-factor-providers>
 
     <dependencies>
-        <owncloud min-version="9.1" max-version="10.0" />
+        <owncloud min-version="10.0" max-version="10.0.99" />

```

Comment:
you can leave max at 10.0 as well - this means any version with 10.0.x does work